ui/src/app/features/breadcrumbs.container/breadcrumbs.container.component.html

```diff
@@ -0,0 +1,22 @@
+<!--
+  ~ Copyright (c) 2020 the original author or authors
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+  ~ or implied. See the License for the specific language governing
+  ~ permissions and limitations under the License.
+  -->
+<div>
+    <app-breadcrumbs
+    [model]="model$ | async"
```

Comment:
Good, this is what I was talking about.
Minor improvement: if the container component template is just the small presentational component declaration, it makes sense to declare  it inside the container component 
```
template:`<app-breadcrumbs
    [model]="model$ | async"
...` 
```

Reply:
then breadcrumbs.container.component.html  can be removed. As well as other empty files like breadcrumbs.container.component.scss